### ISSUE TITLE
Unbundle gramex.ml.r

### ISSUE THREAD

**Issue by sanand0** [gramener/gramex#358]
Raised by @pratapvardhan 

- Should `gramex.ml.r` move to `gramex.r`? It needn't be tied to `ml`.
- Perhaps also bundle it out separately? I might want to use `gramex.r` without needing a complete gramex install.

**Comment by sanand0** [comment]
Agreed on gramex.ml.R. It can be a simple independent package (e.g. “pip install gramex-r”) that’s also imported under gramex.ml. 

@pratapvardhan are you likely to use it in a month or so?
If yes, I’ll get it done by Feb.
If no, I’ll add it to the roadmap and we’ll take it up when we unbundle other packages.

**Comment by jaidevd** [comment]
+1 on making it an independent package.

**Comment by pratapvardhan** [comment]
@sanand0 -- was thinking of playing with gramex.r integration with other frameworks. Not urgent, (if it happens within a month, that'd be great, but can wait) I'll use it whenever it happens.
